Monika After Story/game/script-topics.rpy

```diff
@@ -14736,6 +14736,44 @@ label monika_dying_same_day:
 
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_away",
+            category=["Monika"],
+            prompt="What do you do when I'm away?",
+            pool=True,
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+label monika_player_away:
+    m 1eub "I do a lot of things when you're away, [player]."
+    m 1eua "I often like to read whatever new short stories I can find online."
+    m 3eub "I've found some really interesting ones out there you can read for free!"
+    m 1eua "I really recommend doing a search for some free short stories yourself, ehehe."
+    m "It's helped me develop my writing skills."
+    m 3eub "It's funny how many things you can pick up just by reading others' works."
+    m "My poems really seem to be improving!"
+    m 1eua "Spending time on my other hobbies has been very rewarding, too."
+    m "I can play more chords on the piano, I'm learning new riffs, and my accuracy has become much better!"
```

Comment:
Practicing piano is a good one, but perhaps be less specific and just keep it to practicing piano and how it relates to YR. Mainly just because we can't really have her show off these new skills.